docs/packaging/updating-an-existing-package.md

```diff
@@ -33,24 +33,41 @@ git pull
 
 ## Switch to a New Git Branch
 
-It's always a good idea to switch to a new git branch before beginning packaging work. This will allow you to more easily separate your work from any new changes made to the package repository, which will allow you to more easily rebase any changes if needed.
-Example:
+It's always a good idea to switch to a new git branch before beginning packaging work. This helps to separate your work from any new changes made to the package repository, which will allow you to more easily rebase any changes if needed. To do so run:
 
 ```bash
 git switch -c update_nano
 ```
 
-## Bumping a Package
+## Updating a package
```

Comment:
Maybe it's a good idea to keep Package uppercase so it is is similar to the convention in the rest of the article?

Reply:
Ah, yes. Didn't notice I did that. Thanks!